Makefile

```diff
@@ -192,10 +192,11 @@ generate-client-set:
 
 GIT_COMMIT=$(shell git rev-parse --short HEAD)-dev
 OPERATOR_IMAGE ?= rabbitmqoperator/messaging-topology-operator
+GOFIPS140 ?= off
 .PHONY: docker-build-dev
 docker-build-dev:
 	$(call check_defined, DOCKER_REGISTRY_SERVER, URL of docker registry containing the Operator image (e.g. registry.my-company.com))
-	$(BUILD_KIT) buildx build --build-arg=GIT_COMMIT=$(GIT_COMMIT) -t $(DOCKER_REGISTRY_SERVER)/$(OPERATOR_IMAGE):$(GIT_COMMIT) .
+	$(BUILD_KIT) buildx build --build-arg=FIPS_MODE=$(GOFIPS140) --build-arg=GIT_COMMIT=$(GIT_COMMIT) -t $(DOCKER_REGISTRY_SERVER)/$(OPERATOR_IMAGE):$(GIT_COMMIT) .
```

Comment:
Is this compatible with the case `$GOFIPS140=only`?

Reply:
To verify: `make run GOFIPS140=off GODEBUG='fips140=only'`

Results in:

```
[...]
KUBECONFIG=/Users/acedres/.kube/config OPERATOR_NAMESPACE=rabbitmq-system ENABLE_DEBUG_PPROF=true go run ./main.go -metrics-bind-address 127.0.0.1:9782 --zap-devel
2025-05-27T13:31:42+01:00	INFO	setup	FIPS 140-3 mode enabled
2025-05-27T13:31:42+01:00	INFO	setup	starting manager
2025-05-27T13:31:42+01:00	INFO	controller-runtime.metrics	Starting metrics server
[...]
```